bootstrapvz/base/bootstrapinfo.py

```diff
@@ -55,6 +55,13 @@ def __init__(self, manifest=None, debug=False):
 		# that hint at how a command may be made available.
 		self.host_dependencies = {}
 
+		# Dictionary to store information about the image, like its resulting name
+		image_name = self.__parse_image_name(self.manifest, self.manifest_vars)
+		self.image = {
+		    'filename': image_name + '.' + self.volume.extension,
```

Comment:
EBS backed volumes don't have an extension and the vagrant plugin uses `.box` as an extension, which is not a volume in itself. It it wise to lock in those things here? Wouldn't it be better to just define the dict and let other tasks handle the dict population?